Misc/NEWS.d/next/Library/2024-06-11-07-17-25.gh-issue-119180.iH-2zy.rst

```diff
@@ -0,0 +1,4 @@
+As part of implementing :pep:`649` and :pep:`749`, add a new module
+``annotationlib``. Add support for unresolved forward references in
```

Comment:
```suggestion
:mod:`annotationlib`. Add support for unresolved forward references in
```

Reply:
This doesn't work yet because this PR doesn't add documentation for the new module. I'll do that in a separate module.